pom.xml

```diff
@@ -138,6 +133,30 @@
       <version>2.30</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
Why are these new dependencies now required? (`matrix-auth` and `matrix-project`)